README.md

```diff
@@ -58,6 +58,7 @@ Default implementation [`lib/fetch-template.js`](./lib/fetch-template.js) fetche
 * `templatesPath` - To specify the path where the templates are stored locally, Defaults to `/templates/`
 * `fragmentTag` - Name of the fragment tag, defaults to `fragment`
 * `handledTags` - An array of custom tags, check [`tests/handle-tag`](./tests/handle-tag.js) for more info
+* `maxTemplates` - By default, it equals `20`. You can limit templates cache size which you want to keep, it relates to how many templates you have in your app.
```

Comment:
After some thinking I came to the conclusion that this option should be `0` by default and so memoization should be disabled. As misuse of this parameter in conjunction with default value of `20` may lead to poorer performance & high memory consumption.